transport/tchannel/channel_transport.go

```diff
@@ -82,11 +82,12 @@ func (options transportOptions) newChannelTransport() *ChannelTransport {
 		logger = zap.NewNop()
 	}
 	return &ChannelTransport{
-		once:   lifecycle.NewOnce(),
-		ch:     options.ch,
-		addr:   options.addr,
-		tracer: options.tracer,
-		logger: logger,
+		once:             lifecycle.NewOnce(),
+		ch:               options.ch,
+		addr:             options.addr,
+		tracer:           options.tracer,
+		logger:           logger,
+		forwardingHeader: options.forwardingHeader,
```

Comment:
This is called forwardingHeader here but rawHeader elsewhere. Please use the same name.